### DISCUSSION TITLE
Is django-cms a CMS (Content Management System)?

### DISCUSSION THREAD

**Discussion by olariuromeo** [django-cms/django-cms#7875] [General]
According to [wikipedia](https://en.wikipedia.org/wiki/Content_management_system) CMS means: 
_"A content management system (CMS) is computer software used to manage the creation and modification of digital content (content management). "_ ...etc

... but I don't understand why this DJANGO module (djangp-cms) is so misleading name, because its name has nothing to do with what it does, djangp-cms is not even able to manage its documentation, 

in fact it uses Sphinx or readthedocs for its own documentation, so if django-cms is not capable of managing even some of its documentation text withe the django-cms module, how can DIVIO claim to use this name django-cms?  what is the truth or purpose? 

It seems that its purpose is only to attract some customers to the divio cloud?
Which is otherwise very expensive compared to what the cloud market ofer, so it makes no sense, except that it brings them some customers. Sem to be only a  trap in order to atract some customer, in fact, all the source code refers to divio cloud.

Can someone explain to me why this module has this name?

**Comment by jrief** [comment]
Django-CMS is a very capable **C**ontent **M**anagement **S**ystem used by many organizations and it can handle thousands of pages in dozen of languages, just as you would expect from any decent CMS.

Keeping the documentation as Restructured Text on ReadTheDocs is btw. a very good idea because it keeps the documentation inside the same repository as used for the source code of the CMS. This is to ensure that each version of the CMS corresponds to the same version of the documentation. Ever heard of atomic commits? That concept is not only used by django-CMS but by most applications and libraries, I am aware of.

**Comment by marksweb** [comment]
> ... but I don't understand why this DJANGO module (djangp-cms) is so misleading name, because its name has nothing to do with what it does, djangp-cms is not even able to manage its documentation, 
> 
> in fact it uses Sphinx or readthedocs for its own documentation, so if django-cms is not capable of managing even some of its documentation text withe the django-cms module, how can DIVIO claim to use this name django-cms?  what is the truth or purpose? 

It is industry standard to manage documentation with sphinx. And readthedocs is used by the vast majority of python packages. 
If the documentation was managed by the cms itself, it would cease to be open source because it'd be stored in a private database. 


> It seems that its purpose is only to attract some customers to the divio cloud?

I'm not sure what gives you this impression. Divio originally authored the package, but now they're one of many organisations that use it. They also allow you to run many other packages on their cloud.

Perhaps your issue and reason for posting this is with Divio and not us? 

> Sem to be only a  trap in order to atract some customer, in fact, all the source code refers to divio cloud.

Could you please reference these? I've been a user of django-cms for 10 years and django cms association technical lead for maybe 3. I've always hosted with AWS and not noticed references all over the source code.

> Can someone explain to me why this module has this name?

Because it's a content management system for Django.

**Comment by diab88** [comment]
Yes, django CMS is indeed a Content Management System (CMS). It enables users to create, edit, and publish digital content on a user-friendly platform that supports a diverse range of media and text formats. It simplifies content management processes, making it accessible not just to developers but also to content editors (non-technical users).

Divio has played a pivotal role in the development of django CMS, having been its creator and main contributor for over a decade. Divio's involvement has been deeply rooted in advancing the platform's capabilities and ensuring its alignment with the latest technological standards and user expectations. Even after the transfer of django CMS to an independent association in 2021, Divio has remained actively involved in its evolution, reflecting our commitment to supporting and nurturing the open-source community.

In addition to our history of dedication to django CMS, Divio now offers comprehensive cloud hosting and management solutions. These are designed for a wide array of containerized applications, providing scalable, secure, and efficient cloud hosting environments not limited to django CMS applications. Our extensive experience with django CMS enriches our expertise, enabling us to offer unique insights and tailored services that benefit all our clients in the cloud computing space.

In summary, django CMS is a non-profit organization that supports the community in developing and maintaining an open-source content management system (CMS). Divio is a separate company that offers cloud hosting and cloud management solutions enriched with developer tools and development workflows for containerized web applications. You don't have to use Divio if you want to run your django CMS application. However, if you choose to do so, you can set up and run your django CMS application easily in minutes and deploy it to the cloud without all the DevOps clutter.

If you would like to know more about django CMS or Divio, please feel free to reach out to us at ( info@divio.com ).
